xun/tests/test_stores.py

```diff
@@ -26,7 +26,9 @@ def Memory():
 @contextmanager
 def TmpDiskCache():
     with tempfile.TemporaryDirectory() as tmpdirname:
-        yield xun.functions.store.DiskCache(tmpdirname)
+        ctx = xun.functions.store.DiskCache(tmpdirname)
+        yield ctx
+        ctx.driver.cache.close()
```

Comment:
Delete DiskCache, remember to remove the dependency too :D

Reply:
DiskCache removed, also from dependencies :)